index.html

```diff
@@ -10,6 +10,14 @@
     <script defer src="script.js"></script>
   </head>
   <body>
+    <ul>
+      <li><a class="active" href="index.html">Home</a></li>
+      <li><a href="updates.html">Updates</a></li>
```

Comment:
Bad link. Please commit a updates.html file in the next commit.

In fact, please delete this entirely. It's already in a collapsible area that you didn't delete.